docs/plugins.md

```diff
@@ -263,13 +263,16 @@ in the `tool.poetry.requires-plugins` section of the pyproject.toml file:
 ```toml
 [tool.poetry.requires-plugins]
 my-application-plugin = ">1.0"
+custom-plugin = {path = "custom_plugin/src", develop = true}
```

Comment:
```suggestion
custom-plugin = {path = "custom_plugin", develop = true}
```

Normally, you use the project path, no matter if the project uses source layout or flat layout. It tried with poetry-plugin-export and it seems to work with src.